filebeat/README.md

```diff
@@ -170,6 +171,28 @@ namespace which gives it access to the host loopback device, services listening
 on localhost, could be used to snoop on network activity of other pods on the
 same node.
 
+### How to change readinessProbe for outputs which don't support testing
+
+Some [Filebeat outputs][] like [Kafka output][] don't support testing using
+`filebeat test output` command which is used by Filebeat chart readiness probe.
+
+This makes Filebeat pods crash before being ready with the following message:
+`Readiness probe failed: kafka output doesn't support testing`.
+
+The workaround when using this kind of output is to override the readiness probe
+command to check Filebeat API instead (same as existing liveness probe).
+
+```
+readinessProbe:
+  exec:
+    command:
+      - sh
+      - -c
+      - |
+        #!/usr/bin/env bash -e
+        curl --fail 127.0.0.1:5066
```

Comment:
I'm not sure why this is wrapped in `sh` and then `bash`, I think it could just be
```
readinessProbe:
  exec:
    command:
      - curl
      - --fail
      - 127.0.0.1:5066
```
But I do see this matches the existing probes so it's a discussion for somewhere else.